### PR TITLE
volunteers need id on the token

### PR DESCRIPTION
**Pull request by noam-almog**
adding property on login token

### CODE REVIEW

**Comment by ArtyomKa** [comment]
Where is this userid coming from? please see #398 for details.

**Comment by noam-almog** [comment]
related to issue #398

File: routes/main_routes.js [hunk 1]
```diff
@@ -85,13 +86,16 @@ module.exports = function (app, passport) {
...
                 } else {
-                    res.cookie('authToken', passportLib.generateJwtToken(req.body.email), {httpOnly: true, domain:'.midburn.org'});
-                    var r = req.body['r'];
-                    if (r) {
-                        return res.redirect(r);
-                    } else {
-                        return res.redirect('home');
-                    }
+                    console.log('\n\n\nget_user_by_email');
+                    return DrupalAccess.get_user_by_email(req.body.email)
```
Comment (noam-almog) [inline routes/main_routes.js:90]: @ArtyomKa @LeonFedotov is that what you guys were thinking about ?

**Comment by ArtyomKa** [comment]
@noam-almog I think this pr should be closed.

**Pull request closed without merging**